pkg/cli/longhorn.go

```diff
@@ -139,12 +158,377 @@ func NewLonghornPrepareForMigration(_ CLI) *cobra.Command {
 				log.Printf("$ kurl longhorn rollback-migration-replicas")
 				log.Printf("")
 			}
-			log.Print("All Longhorn volumes and nodes are healthy, ready for migration.")
+			log.Print("All Longhorn volumes and nodes are healthy.")
+
+			if err := scaleDownPodsUsingLonghorn(cmd.Context(), cli); err != nil {
+				return fmt.Errorf("error scaling down pods using longhorn volumes: %s", err)
+			}
+			log.Print("Environment is ready for the Longhorn migration.")
 			return nil
 		},
 	}
 }
 
+// scaleUpPodsUsingLonghorn scales up any deployment or statefulset that has been previously
+// scaled down by scaleDownPodsUsingLonghorn. uses the default annotation used by pvmigrate.
+func scaleUpPodsUsingLonghorn(ctx context.Context, cli client.Client) error {
+	if err := scaleEkco(ctx, cli, 1); err != nil {
+		return fmt.Errorf("error scaling ekco operator back up: %w", err)
+	}
+	if err := scalePrometheus(ctx, cli, 2); err != nil {
```

Comment:
does it make sense here to use the desired scale annotation? what if we make a decision in the future to use only one replica for single node installations?

Reply:
Fixed.